### PR TITLE
Dynamic number of bins with limits

### PR DESCRIPTION
**Pull request by bilgili**

### CODE REVIEW

File: lexis/render/histogram.fbs [hunk 1]
```diff
@@ -2,17 +2,20 @@
 //                     Ahmet Bilgili <ahmet.bilgili@epfl.ch>
 
 // This event is used to communicate histograms for
-// 8 bit data between applications ( also inside the
-// application ).
+// variable number of bins
```
Comment (hernando) [inline lexis/render/histogram.fbs]: histograms with arbitrary number of bins.

File: lexis/render/histogram.fbs [hunk 2]
```diff
@@ -2,17 +2,20 @@
...
 //
-// Currently every data is discretized to 256 bins.
-// This may change later to higer number of bins if
-// needed.
+// Data values between max and min are discretized into
+// the size of the bins.
```
Comment (hernando) [inline lexis/render/histogram.fbs]: This sentence sounds strange to me, but I don't have a good suggestion.

File: lexis/render/histogram.fbs [hunk 3]
```diff
@@ -1,18 +1,21 @@
...
-// 8 bit data between applications ( also inside the
-// application ).
+// This event is used to communicate histograms with
+// arbitrary number of bins.
 //
-// Currently every data is discretized to 256 bins.
-// This may change later to higer number of bins if
-// needed.
+// Data values between max and min are discretized into
+// number of the bins.
```
Comment (hernando) [inline lexis/render/histogram.fbs]: A histogram divides a range [min, max] of possible values in n bins and counts how many elements fall inside each bin.

Comment (hernando) [inline lexis/render/histogram.fbs]: "into number of the bins" sounds strange.

Comment (bilgili) [reply lexis/render/histogram.fbs]: You made the definition of the histogram :)

Comment (hernando) [reply lexis/render/histogram.fbs]: Yes, but I don't know what else you could write here.

**Comment by hernando** [comment]
Are you going to change it?

**Comment by bilgili** [comment]
Why not :)

**Comment by bilgili** [comment]
Where is my +1 ? :)

**Comment by bilgili** [comment]
retest this please

**Comment by hernando** [comment]
+1

**Pull request merged**